content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -477,17 +478,19 @@ are different ways to arrange for pods to work on different things.
 This table shows the required settings for `.spec.parallelism` and `.spec.completions` for each of the patterns.
 Here, `W` is the number of work items.
 
-|             Pattern                       | `.spec.completions` |  `.spec.parallelism` |
-| ----------------------------------------- |:-------------------:|:--------------------:|
-| [Queue with Pod Per Work Item]            |          W          |        any           |
-| [Queue with Variable Pod Count]           |         null        |        any           |
-| [Indexed Job with Static Work Assignment] |          W          |        any           |
-| [Job Template Expansion]                  |          1          |     should be 1      |
+|             Pattern                             | `.spec.completions` |  `.spec.parallelism` |
+| ----------------------------------------------- |:-------------------:|:--------------------:|
+| [Queue with Pod Per Work Item]                  |          W          |        any           |
+| [Queue with Variable Pod Count]                 |         null        |        any           |
+| [Indexed Job with Static Work Assignment]       |          W          |        any           |
+| [Job Template Expansion]                        |          1          |     should be 1      |
+| [Intra-job Pod Networking Using Pod Hostnames]  |         any         |        any           |
```

Comment:
it's not the number of "work items", but it's the number of workers. Not sure if this proposal makes the table more confusing. The important thing it I want to highlight is that parallelism=completions in this case.

```suggestion
| [Intra-job Pod Networking Using Pod Hostnames]  |         W         |        W           |
```

Reply:
Done